travis/travis_install_nightly

```diff
@@ -3,12 +3,14 @@ set -v
 
 if [ "${LINT_CHECK}" != "0" ]; then
     if [ -f "${HOME}/.nvm/nvm.sh" ]; then
+        echo "Configuring the nvm"
         # Update nodejs v6.latest required by eslint
         # Using nvm of travis
         CURRENT_NODE=$(which node)
         source ${HOME}/.nvm/nvm.sh
         nvm install 6
         ln -sf $(nvm which 6) $CURRENT_NODE
+        echo "source $HOME/.nvm/nvm.sh" >> ${HOME}/.bashrc
```

Comment:
Do you try add directly this `source` command for our bash profile from big image?